content/en/platform-engineering.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Platform Engineering
+status: Feedback Appreciated
+category: ["methodology", "platform"]
+---
+
+Platform engineering is the art of building tools and processes that empower software developers.  
+It's like creating a self-service station specifically designed to meet developers' needs throughout the 
+software creation journey. By providing these tools and workflows, platform engineering helps developers 
+to work faster and more efficiently.
+
+## Problem it addresses
+
+Platform engineering tackles the challenge of slow and inefficient software development. 
+In the past, developers often spent a significant amount of time on repetitive tasks like setting up 
+environments or configuring tools. Platform engineering streamlines these processes, 
```

Comment:
here I would love to see the connection to the infrastructure.

Reply:
Hi @iamNoah1 , updated the file as per your suggestion, kindly check and review.